package.json

```diff
@@ -156,6 +171,10 @@
     "connected-react-router": "^6.4.0",
     "debug": "^4.0.1",
     "font-awesome": "^4.7.0",
+    "glob": "^7.1.4",
+    "html-loader": "^0.5.5",
+    "html-webpack-plugin": "^3.2.0",
+    "i": "^0.3.6",
```

Comment:
Why are `html-webpack-plugin`, `html-loader`, and `i` added in this PR? (sorry if I am missing something obvious ... I can see that `glob` is introduced by this PR, but I do not see the reason for the other NPM packages)

Reply:
Mhmm, some of these packages already exist in the `devDependencies`:
https://github.com/readium/readium-desktop/blob/bb36f5b545e351654ed07fef0f55fde635de5c01/package.json#L234-L236
...but in this PR they are added to `dependencies`, there is something wrong here!
(and what about `i`?)

Reply:
Ah, I now understand. The recent sync you did in this PR's branch with the develop branch wasn't quite correct. These 4 packages must be removed from `dependencies` (in this PR). See: https://github.com/readium/readium-desktop/pull/504

Reply:
done